cabot/metricsapp/tests/fixtures/elastic/grafana/query_builder/grafana_series_order_sub_agg_query.json

```diff
@@ -38,18 +43,13 @@
             "min_doc_count": 1
           },
           "aggs": {
-            "1": {
-              "avg": {
-                "field": "timing"
-              }
-            },
             "agg": {
               "date_histogram": {
                 "field": "@timestamp",
                 "interval": "20m",
                 "extended_bounds": {
-                  "min": "now-100m",
-                  "max": "now"
+                  "max": "now",
+                  "min": "now-100m"
```

Comment:
Why did this change??

Reply:
Probably Python's JSON serializer being nondeterministic. I just copied over the new JSON object in its entirety.